backend/src/types.ts

```diff
@@ -1312,3 +1315,9 @@ export type AuthKind = K8sResourceCommon & {
     allowedGroups: string[];
   };
 };
+
+export enum OdhPlatformType {
+  OPEN_DATA_HUB = 'Open Data Hub',
+  SELF_MANAGED_RHOAI = 'OpenShift AI Self-Managed',
+  MANAGED_RHOAI = 'OpenShift AI Cloud Service',
+}
```

Comment:
can you add a comment here referencing where you got the values for future reference

https://github.com/red-hat-data-services/rhods-operator/blob/main/pkg/cluster/const.go